### ISSUE TITLE
Prioritize a specific task

### ISSUE THREAD

**Issue by real-or-random** [cirruslabs/cirrus-ci-web#450]
A possibility to move a specific queued task (or some tasks) to the front of the queue will be very helpful from time to time. For example, when you made a PR that fixes an issue on MacOS but all the MacOS builds come last in the queue.

**Comment by fkorotkov** [comment]
Duplicate of https://github.com/cirruslabs/cirrus-ci-docs/issues/525

**Comment by real-or-random** [comment]
Ah nice! Though I don't think it's a 100% duplicate. I was not thinking about a config option but rather about some button in the web interface, to move a task only once.
